frontend/packages/console-plugin-shared/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@console/plugin-shared",
+  "version": "0.0.0-fixed",
+  "description": "Console plugin shared code",
+  "private": true,
```

Comment:
we will need a script which will generate a publish-able npm package. We do the same for sdk https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/package.json#L11

Reply:
Let's use `private: false` and avoid generating package.json